### PR TITLE
uses tmpreaper instead of role celery_hq

### PR DESCRIPTION
**Pull request by javierwilson**

### CODE REVIEW

**Comment by javierwilson** [comment]
this requires #807 , once that is merged, i will test again...

File: ansible/deploy_commcarehq.yml [hunk 1]
```diff
@@ -16,4 +16,6 @@
     - celery
   sudo: yes
   roles:
-    - {role: celery_hq, tags: celery_hq}
+    - role: tmpreaper
+      tags: commcarehq, celery_hq, tmpreaper, cron
+      tmpreaper_time: '2d'
```
Comment (gcapalbo) [inline ansible/deploy_commcarehq.yml]: do we need to specify the directory or does it only operate on /tmp?

**Comment by gcapalbo** [comment]
:+1: looks good to me. so just looks like we're replacing the cron job with tmpreaper to clear tmp files?

also just curious if this has been tested

File: ansible/deploy_commcarehq.yml (hunk 1 above)
Comment (javierwilson) [inline ansible/deploy_commcarehq.yml]: yes, this works in /tmp and /var/tmp, we can add more dirs if neede

**Pull request merged**